pkg/controller/storagecluster/initialization_reconciler.go

```diff
@@ -205,9 +208,14 @@ func (r *ReconcileStorageCluster) ensureExternalStorageClusterResources(instance
 				// Setting the PoolName for RBD StorageClass
 				sc = scs[1]
 			} else if d.Name == cephRgwStorageClassName {
+				rgwEndpoint := d.Data[externalCephRgwEndpointKey]
+				extCephObjectStores, err = r.newExternalCephObjectStoreInstances(instance, rgwEndpoint, reqLogger)
+				if err != nil {
+					return err
+				}
 				// Set the external rgw endpoint variable for later use on the Noobaa CR (as a label)
 				// Replace the colon with an underscore, otherwise the label will be invalid
-				externalRgwEndpointReplaceColon := strings.Replace(d.Data[externalCephRgwEndpointKey], ":", "_", -1)
+				externalRgwEndpointReplaceColon := strings.Replace(rgwEndpoint, ":", "_", -1)
 				externalRgwEndpoint = externalRgwEndpointReplaceColon
 
 				// Setting the Endpoint for OBC StorageClass
```

Comment:
We don't need to set the endpoint anymore on the SC for OBC. We just need to set the `objectStoreName` back to it, just like we do for the converged case.

Reply:
Made the changes, @leseb , please take a look...